src/cdk/table/table.ts

```diff
@@ -464,16 +495,13 @@ export class CdkTable<T> implements AfterContentChecked, CollectionViewer, OnDes
   @Output()
   readonly contentChanged = new EventEmitter<void>();
 
-  // TODO(andrewseguin): Remove max value as the end index
-  //   and instead calculate the view on init and scroll.
   /**
    * Stream containing the latest information on what rows are being displayed on screen.
    * Can be used by the data source to as a heuristic of what data should be provided.
    *
    * @docs-private
    */
-  readonly viewChange =
-      new BehaviorSubject<{start: number, end: number}>({start: 0, end: Number.MAX_VALUE});
+  readonly viewChange: BehaviorSubject<ListRange>;
```

Comment:
Would it make more sense to make this a ReplaySubject so that users will get the latest value on subscribe but you don't need to set an initial value?

Reply:
Changing this to a ReplaySubject would be a breaking change. Is it alright to make that change in this  PR?